version/constants.go

```diff
@@ -12,9 +12,12 @@ import (
 	"github.com/ava-labs/avalanchego/utils/constants"
 )
 
-// RPCChainVMProtocol should be bumped anytime changes are made which require
-// the plugin vm to upgrade to latest avalanchego release to be compatible.
-const RPCChainVMProtocol uint = 29
+const (
+	Client = "avalanche"
```

Comment:
I'd prefer for the client name passed around in the new version message to be `avalanchego` rather than `avalanche`

Reply:
I originally planned on doing this in a separate PR but I'm down to do it as part of this one